tests/hikari/impl/test_entity_factory.py

```diff
@@ -3081,6 +3089,9 @@ def test_deserialize_command_interaction(
             41771983423143936: entity_factory_impl.deserialize_role(guild_role_payload, guild_id=43123123)
         }
         assert interaction.resolved.users == {115590097100865541: entity_factory_impl.deserialize_user(user_payload)}
+
+        assert interaction.resolved.messages == {123: entity_factory_impl.deserialize_message(message_payload)}
```

Comment:
More of a remark than anything, the spacing should be removed here
```suggestion
        assert interaction.resolved.messages == {123: entity_factory_impl.deserialize_message(message_payload)}
```